api/build.gradle

```diff
@@ -19,7 +19,14 @@ sourceSets {
 }
 
 jar {
-    manifest {
-        attributes('Automatic-Module-Name': 'org.reactivestreams')
-    }
+    bnd ('Bundle-Name': 'reactive-streams-jvm',
+            'Bundle-Vendor': 'Reactive Streams SIG',
+            'Bundle-Description': 'Reactive Streams API',
+            'Bundle-DocURL': 'http://reactive-streams.org',
+            'Bundle-Version': project.version,
+            'Import-Package': '!org.junit,!junit.framework,!org.mockito.*,!org.testng.*,*',
```

Comment:
Are these needed for all the jars?

Reply:
It used to be an import bloat with OSGi due to it including test dependencies. I'll double check.

Reply:
Yep, they are no longer necessary, plus the TestNG imports have to be named for the TCK bundle anyway.